packages/blockchain-link-types/src/blockbook-api.ts

```diff
@@ -47,7 +47,7 @@ export interface TokenTransfer {
     to: string;
     contract: string;
     name: string;
-    symbol: string;
+    symbol?: string;
```

Comment:
this is autogenerated file from bb which is not supposed to be changed by hand. cc @martinboehm is it valid case that it might come like this? and if yes, it looks gen script did not work as expected.

Reply:
I believe all the attributes should reflect reality ideally.

Reply:
let's try to be real but not necessarily in one step